### ISSUE TITLE
Provide ability to deploy GM Data without deploying Mongo

### ISSUE THREAD

**Issue by chrisbsmith** [DecipherNow/helm-charts#472]
Specifically, we need the ability to deploy a GM Data instance and point it to an existing Mongo DB.  The currently charts always expect to deploy a mongodb and configure GM Data to use it.

**Comment by rfielding** [comment]
MONGOHOST can be set to point to a Mongo.  We need an env var to not deploy a Mongo?

**Comment by rfielding** [comment]
and... is this a big mongo cluster?  MONGOHOST can be of the form of a list of host:port, like: `mdb1:27017,mdb2:27017`

**Comment by chrisbsmith** [comment]
Yes, we define MONGOHOST but its battle value is calculated based on other inputs. We simply need to move that field up and allow for the user to set it
